act-rules-format.bs

```diff
@@ -403,10 +416,10 @@ Definitions {#definitions}
       <li>**Failed**: The test subject does not meets all expectations</li>
     </ul>
     <div class="note">
-      <p>While *inapplicable* is a valid outcome for ACT Rules, it might not be a valid result for all [accessibility requirements](#accessibility-requirements). Notably the success criteria of WCAG 2.0 and WCAG 2.1 can only be evaluated to satisfied (passed and inapplicable) or not satisfied (failed).</p>
+      <p><strong>Note</strong>: While *inapplicable* is a valid outcome for ACT Rules, it might not be a valid result for all [accessibility requirements](#accessibility-requirements). Notably the success criteria of WCAG 2.0 and WCAG 2.1 can only be evaluated to satisfied (passed and inapplicable) or not satisfied (failed).</p>
     </div>
     <div class="note">
-      <p>Implementers using the [[EARL10-Schema]] can express the outcome on the [outcome property](https://www.w3.org/TR/EARL10-Schema/#outcome). In addition to `passed` `failed` and `inapplicable`, EARL 1.0 also defined an `incomplete` outcome. While this can not be the outcome of an ACT Rule when applied in its entirety, it often happens that rules are only partially evaluated. For example, when applicability was automated, but the expectations have to be evaluated manually. Such "interim" results can be expressed with the `incomplete` outcome.
+      <p><strong>Note</strong>: Implementers using the [[EARL10-Schema]] can express the outcome on the [outcome property](https://www.w3.org/TR/EARL10-Schema/#outcome). In addition to `passed` `failed` and `inapplicable`, EARL 1.0 also defined an `incomplete` outcome. While this can not be the outcome of an ACT Rule when applied in its entirety, it often happens that rules are only partially evaluated. For example, when applicability was automated, but the expectations have to be evaluated manually. Such "interim" results can be expressed with the `incomplete` outcome.
```

Comment:
Same comment, "using the [outcome property]" instead of "on".

Do we have 'incomplete' as a valid result defined in our list of outcomes?

Reply:
No, incomplete is not an outcome recognised in the ACT Rules Format.